openwebwork/webwork2
# WebworkClient Local Storage

This adds a local storage engine to the WebworlClient path for rendering problems.  Specifically it has some new js files and code which stores submitted answers locally using browser local storage so that they can persist when the same problem is brought up again.  

To test: 
1.  Get the pull
2.  Change the `outputformat` in `clients/TestWW.html` to `sticky`. 
3.  Load TestWW.html and try submitting some answers to problems.  Close the window and open it again.  When you load the problems up the previous answers should still be there (even after a hard refresh).  
4.  Also check the `simple` output format since I added some bootstrap classes to make things render a bit better.
